packages/get-platform/src/__tests__/parseOpenSSLVersion.test.ts

```diff
@@ -72,6 +82,16 @@ describe('parseLibSSLVersion', () => {
       content: `/lib/libssl.so.3.1`,
       expect: '3.0.x',
     },
+    {
+      name: 'libssl.so.0.9.8 is unsupported',
```

Comment:
let’s also add a test for openssl 2.x (it’s based on 0.9 too I think)

Reply:
Done